packaging/rpm/poplog.spec

```diff
@@ -0,0 +1,70 @@
+Name: poplog
+Version: 0.2
```

Comment:
We need to set these from the VERSION file, consider putting some special string here and subbing out with sed when we copy over to the `_build` directory in the make target.